lib/bundler/plumber/scanner.rb

```diff
@@ -118,12 +115,10 @@ def scan_specs(options={})
 
         @lockfile.specs.each do |gem|
           @database.check_gem(gem) do |advisory|
-
-            # TODO this logic should be modified for rubymem
-            #unless (ignore.include?(advisory.cve_id) || ignore.include?(advisory.osvdb_id))
-            #  yield UnpatchedGem.new(gem,advisory)
-            #end
-            yield UnpatchedGem.new(gem, advisory)
+            gem_and_id = "#{advisory.gem}-#{advisory.id}"
+            unless ignore.include?(gem_and_id)
+              yield UnpatchedGem.new(gem,advisory)
+            end
```

Comment:
@bronzdoc Why not just `yield UnpatchedGem.new(gem,advisory) unless ignore.include?(gem_and_id)`?